source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Domain/Dtos/Validation/ValidationRuleIdentifier.cs

```diff
@@ -48,6 +48,8 @@ public enum ValidationRuleIdentifier
         TransparentInvoicingIsNotAllowedForFee = 34, // VR904 / D67
         ChargeResolutionCanNotBeUpdated = 35, // VR907 / D23
         RecipientRoleMustBeDdz = 36, // VR165 / E55
-        NumberOfPointsMatchTimeIntervalAndResolution = 37, // VR909 / E87
+        NumberOfPointsMatchTimeIntervalAndResolution = 37, // VR909 / E87,
+        ChargeNameIsRequired = 38, // VRXXX / E0H
+        ChargeDescriptionIsRequired = 39, // VRXXX / E0H
```

Comment:
I have added them to our internal validation sheet, so please use below.

```suggestion
        ChargeNameIsRequired = 38, // VR910 / E0H
        ChargeDescriptionIsRequired = 39, // VR911 / E0H
```